cmd/pomerium-cli/cache.go

```diff
@@ -10,8 +10,46 @@ import (
 	"os"
 	"path/filepath"
 	"time"
+
+	"github.com/spf13/cobra"
 )
 
+var cacheCmd = &cobra.Command{
+	Use:   "cache",
+	Short: "commands for working with the cache",
+}
+
+var cacheClearCmd = &cobra.Command{
+	Use:   "clear",
+	Short: "clear the cache",
+	RunE: func(_ *cobra.Command, _ []string) error {
+		root, err := os.UserCacheDir()
+		if err != nil {
+			return err
+		}
+		return os.RemoveAll(filepath.Join(root, "pomerium-cli"))
```

Comment:
It might be nice to extract this bit of logic to a single shared method. (I see there's already a `cachePath()` method, but maybe we could rename that one to `execCredentialCachePath()`?)

(Totally optional; we can always refactor this in a follow-up PR.)